functions/consumer/sftp-consumer/src/main/java/org/springframework/cloud/fn/consumer/sftp/SftpConsumerConfiguration.java

```diff
@@ -56,8 +59,13 @@ public IntegrationFlow ftpOutboundFlow(SftpConsumerProperties properties,
 		if (properties.getFilenameExpression() != null) {
 			handlerSpec.fileNameExpression(properties.getFilenameExpression());
 		}
+
+		if (sftpMessageHandlerSpecCustomizer != null) {
+			sftpMessageHandlerSpecCustomizer.customize(handlerSpec, "sftpMessageHandler");
```

Comment:
Does "beanName" have to be an actual bean name?

Reply:
No. You have better ideas? 🤗

Reply:
I noticed that the other specs we call `id` and then that is also what we pass into the customizer. If its simply symbolic name this is fine w/ me.

Reply:
We probably need to look at this from end-user perspective to give them an easy way to make a sense from the names we can provide them. 

I’ll give this more thoughts over night…

Reply:
So, the name we provide to the customizer for end-user consideration must really be a real bean name.
Who knows may be they would like to have an access to that bean eventually...

Or... for consistency with Spring Boot customizer API we just can drop the `beanName` argument altogether...

Reply:
If we do not have a use case yet that would require the bean name, I would be in favor of dropping it altogether (until needed). 

1. makes it consistent w/ the regular "customizer" contract (as you pointed out) 
2. makes it a bit simpler